### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by clabnet** [oatpp/oatpp#458]
Hi, 
VS2019, only files in error are related to OAT++ 1.2.5.

How to solve ?
Thank's

```
Build started...
1>------ Build started: Project: licmgrapi-test, Configuration: Debug x64 ------
1>license_controller_test.cpp
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(45,76): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(45): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(45): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ';'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(45,76): error C2059: syntax error: ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(45,77): error C2027: use of undefined type 'oatpp::network::virtual_::<unnamed-tag>'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(57): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(57,17): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(57): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(57): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ';'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(57,17): error C2059: syntax error: ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/Interface.hpp(57,17): error C2027: use of undefined type 'oatpp::network::virtual_::<unnamed-tag>'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(49): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(49,21): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(49): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(49): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ';'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(49,21): error C2059: syntax error: ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(49,21): error C2027: use of undefined type 'oatpp::network::virtual_::<unnamed-tag>'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(56,112): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(56): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(56): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ';'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(56,112): error C2059: syntax error: ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/client/ConnectionProvider.hpp(56,113): error C2027: use of undefined type 'oatpp::network::virtual_::<unnamed-tag>'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(51): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(51,21): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(51): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(51): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ';'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(51,21): error C2059: syntax error: ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(51,21): error C2027: use of undefined type 'oatpp::network::virtual_::<unnamed-tag>'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(58,112): error C2332: 'struct': missing tag name
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(58): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(58): error C2144: syntax error: 'oatpp::network::virtual_::<unnamed-tag>' should be preceded by ';'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(58,112): error C2059: syntax error: ')'
1>C:\Program Files (x86)\oatpp\include\oatpp-1.2.5\oatpp\oatpp/network/virtual_/server/ConnectionProvider.hpp(58,113): error C2027: use of undefined type 'oatpp::network::virtual_::<unnamed-tag>'
1>Done building project "licmgrapi-test.vcxproj" -- FAILED.
2>------ Skipped Build: Project: INSTALL, Configuration: Debug x64 ------
2>Project not selected to build for this solution configuration 
========== Build: 0 succeeded, 1 failed, 12 up-to-date, 1 skipped ==========
```

**Comment by lganzzzo** [comment]
Hello @clabnet ,

The error message looks odd. It's like it doesn't recognize the C++ syntax.
Make sure to compile for the correct C++ version. Min C++ version required for oatpp is `11`

**Comment by lganzzzo** [comment]
Fixed in `1.3.0` see https://github.com/oatpp/oatpp/commit/809b3fb3f16a28e8848362e34e1fc87cfe778dc4
